sources/packages/backend/apps/db-migrations/src/sql/InstitutionLocations/Add-col-has-integration.sql

```diff
@@ -0,0 +1,7 @@
+-- Add has_integration to locations table
+ALTER TABLE
```

Comment:
Is that choice of using integration is at the location level? I was assuming it to be at institution level.

Reply:
The ticket is clear: "Add Boolean column to **institution location** table that will flag if institution uses integration".
I believe that we ar ok 😉

Reply:
Actually my doubt was on the ticket itself 😊. I was trying to understand that why the decision of using the integration is not taken at the institution level and at location level.